gr00t/model/policy.py

```diff
@@ -231,10 +235,19 @@ def _check_state_is_batched(self, obs: Dict[str, Any]) -> bool:
         return True
 
     def _load_model(self, model_path):
-        model = GR00T_N1.from_pretrained(model_path, torch_dtype=COMPUTE_DTYPE)
-        model.eval()  # Set model to eval mode
-        model.to(device=self.device)  # type: ignore
-
+        # decide actual setting
+        if self.attn_implementation_preference == "auto":
```

Comment:
Can remove this if else statement here, and just simply parse in the `attn_implementaion` into it